ssv/pre_consensus_justification.go

```diff
@@ -39,8 +27,8 @@ func (b *BaseRunner) validatePreConsensusJustifications(data *types.ConsensusDat
 		return errors.New("wrong beacon role")
 	}
 
-	if data.Duty.Slot <= highestDecidedDutySlot {
-		return errors.New("duty.slot <= highest decided slot")
+	if qbft.Height(data.Duty.Slot) <= b.QBFTController.Height && b.QBFTController.Height != qbft.FirstHeight {
```

Comment:
if we are on first height (which will never happen I guess) then its ok that the duty height is smaller than the controller height? I understand why its ok that its equal but not lower.

Reply:
If it is small or equal it returns an error.
On first height it can only be equal.

Reply:
you mean if firstHeight would be a number different than 0?
If it will be 1 then we won't need this special condition in all those places in the code